libs/block-editor/src/lib/extensions/components/bubble-menu-link-form/bubble-menu-link-form.component.ts

```diff
@@ -0,0 +1,119 @@
+import {
+    Component,
+    OnInit,
+    Input,
+    ViewChild,
+    ElementRef,
+    AfterViewInit
+} from '@angular/core';
+
+// Editor
+import tippy, { Instance } from 'tippy.js';
+import { Editor, posToDOMRect } from '@tiptap/core';
+
+@Component({
+    selector: 'dotcms-bubble-menu-link-form',
+    templateUrl: './bubble-menu-link-form.component.html',
+    styleUrls: ['./bubble-menu-link-form.component.scss']
+})
+export class BubbleMenuLinkFormComponent implements OnInit, AfterViewInit {
+
+    @ViewChild('form') form: ElementRef;
+    @ViewChild('input') input: ElementRef;
+    @ViewChild('formContainer') formContainer: ElementRef;
+    
+    @Input() editor: Editor;
+
+    public nodeLink = '';
+    public newLink = '';
+    public tippy: Instance;
+
+    ngOnInit(): void {
+        this.editor.on('selectionUpdate', () => this.setTippyPosition());
+    }
+
+    ngAfterViewInit() {
+        this.tippy = tippy(this.formContainer.nativeElement as HTMLElement, {
+            duration: 150,
+            getReferenceClientRect: null,
+            content: this.form.nativeElement,
+            interactive: true,
+            trigger: 'manual',
+            placement: 'bottom',
+            hideOnClick: 'toggle',
+            animation: 'fade',
+            appendTo: this.formContainer.nativeElement
+        });
```

Comment:
For what I understand, tippy should be at the editor level, we just pass to the tippy element, we should not be passing editor to this component, this should be a dumb component.

We should just emit events and at the extension, level do our things.